config/300-eventing-v1alpha1-knativeeventing-crd.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: eventings.operator.knative.dev
+  name: knativeeventings.operator.knative.dev
 spec:
   group: operator.knative.dev
   names:
-    kind: Eventing
-    listKind: EventingList
-    plural: eventings
-    singular: eventing
+    kind: KnativeEventing
+    listKind: KnativeEventingList
+    plural: knativeeventings
+    singular: knativeeventing
     shortNames:
     - ke
```

Comment:
I am now really leaning on to removing this short name. For example, if we have older version of knative eventing operator installed, this short name is occupied by the CRD eventings.operator.knative.dev. After we applying the new CRD knativeeventings.operator.knative.dev also with this same short name, the terrible thing will happen: none of the old or the new CR can be found by the operator controller. If we remove this short name or change it to something else, this issue is gone. @cardil @n3wscott

Reply:
That's a valid concern for me. I'm okay with removing it.